usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -127,6 +127,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/recipient/count/](/api/v2/recipient/count/)|POST| Returns the count of recipents for the given filters |
 |[/api/v2/recipient/duns/<HASH_VALUE\>/](/api/v2/recipient/duns/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/)|GET| Returns a high-level overview of a specific recipient, given its id |
 |[/api/v2/recipient/duns/](/api/v2/recipient/duns/)|POST| Returns a list of recipients in USAspending DB |
+|[/api/v2/recipient/recipient_id/](/api/v2/recipient/recipient_id)|POST| Returns an individual recipient in USAspending DB |
```

Comment:
Should update to match `/api/v2/recipient/duns/<HASH_VALUE\>/` where the URL param value is captured by `<...\>` and all capitalized.

Also, should update `POST` to be `GET`.

Reply:
updated to reflect pattern.